src/openforms/forms/tests/test_copy.py

```diff
@@ -33,7 +33,10 @@ def test_form_copy_with_reusable_definition(self):
         self.assertNotEqual(copied_form.pk, form.pk)
         self.assertNotEqual(copied_form.uuid, str(form.uuid))
         self.assertEqual(copied_form.active, form.active)
-        self.assertEqual(copied_form.registration_backend, form.registration_backend)
+        self.assertEqual(
```

Comment:
Maybe we should test the exact backends  and not the number of them(since we are testing the copy functionality)?Like we do with all the assertions above.

Reply:
I've done a number comparison because both have 0 backends :)
But yeah, its a bit ambiguous. I'll make it more explicit 👍

Reply:
let's make it very clear what the expected result is, both the old and new assertion confuse me in what the actual initial and final state are